facilitator-prep.md

```diff
@@ -0,0 +1,35 @@
+# How to prep to facilitate a storytelling session
+
+Welcome to the very important role of facilitating storytelling sessions.
+
+## 1. Confirm your schedule with a storyteller.
+
+* Keep your availability in the team schedule organizer up to date. This is the calendar which storytellers will use to schedule sessions with you.
+
+* Check any other obligation calendars against the scheduled spot your storyteller has signed up for.
+
+* Life happens and schedules change. Communicate as early as possible with your storyteller and our project leads if you need to reschedule or cancel a session.
+
+## 2. Do your facilitation homework.
+
+* Create a copy of [Storyteller Session Prep](https://github.com/opensourcestories/storytelling/session-prep-template.md).
+
+  * Coordinate with storyteller to fill out personal information in the prep material.
+
+  * Choose your Flow from [Conversation Flow for Open Source Storytellers](bit.ly/opensource-storytelling-conversation-flow)
```

Comment:
We should probably update this doc to reflect that sessions are now half the time.

Reply:
Updated w/ 
* new question ratio suggestions (15-mins + 30-mins)
* split out the intro + warm-up question
* started a section for warm-up question
* created [a new issue in story-questions]((https://github.com/opensourcestories/story-questions/issues/3)) to remind myself we need that bank